index.html

```diff
@@ -16,9 +21,7 @@
     <div class="header-container">
         <div class="mobile-version-nav">
             <div class="close-button">
-                <button class="close-button">
-                    <span aria-hidden="true">&times;</span>
-                </button>
+                <i class="fa-solid fa-xmark"></i>
             </div>
             <ul class="mobile-menu-items">
```

Comment:
### OPTIONAL 

- [ ] Kindly remove the blur from the mobile  header and hero section when any of the mobile nav links is clicked, see video below ⬇️  

https://user-images.githubusercontent.com/69638013/168870449-e2cc9c39-9964-4513-a5cb-03d5a72df530.mp4